docs/source/developapps/transactionhandler.md

```diff
@@ -85,7 +85,7 @@ shows you the exact form of these handlers.
 
 Once a handler has been added to the smart contract, it will be invoked during
 transaction processing. During processing, the handler receives `ctx`, the
-[transaction context](./transationcontext.md), performs some processing, and
+[transaction context](./transationcontext.html), performs some processing, and
```

Comment:
fix typo of './transationcontext.html as 'transaction has a typo breaking the link.

Reply:
Thanks for finding. Fixed.